e2e/worker/WORKSPACE

```diff
@@ -22,9 +22,10 @@ load("@aspect_rules_js//npm:npm_import.bzl", "npm_translate_lock")
 
 npm_translate_lock(
     name = "npm",
+    lifecycle_hooks_no_sandbox = False,
```

Comment:
Why was this required?

Reply:
It was. One of the @nestjs lifecycle hooks fails outside of the sandbox. Didn't look further into why. I'll comment there.